core/src/main/java/ai/timefold/solver/core/impl/score/director/easy/EasyScoreDirector.java

```diff
@@ -28,9 +29,9 @@ public final class EasyScoreDirector<Solution_, Score_ extends Score<Score_>>
     private final EasyScoreCalculator<Solution_, Score_> easyScoreCalculator;
 
     public EasyScoreDirector(EasyScoreDirectorFactory<Solution_, Score_> scoreDirectorFactory,
-            boolean lookUpEnabled, boolean constraintMatchEnabledPreference, boolean expectShadowVariablesInCorrectState,
+            boolean lookUpEnabled, boolean expectShadowVariablesInCorrectState,
```

Comment:
It seems that changing the `ConstraintMatchPolicy` is no longer possible. Should it be allowed?

Reply:
Easy has no support for constraint matches, so there is really no point for setting that policy from the outside.